### PR TITLE
Modify plot_friends.py

### PR DESCRIPTION
**Pull request by techytushar**
* Add compatibility with JSON data
* Improve gitignore

Signed-off-by: Tushar Mittal <chiragmittal.mittal@gmail.com>

### CODE REVIEW

File: plot_friends.py [hunk 1]
```diff
@@ -86,5 +69,13 @@ def friends():
     plt.legend(loc='upper left', ncol=2)
     plt.show()
 
+    print('\nSome statistics based on the data: ', maxdays)
```
Comment (kaustubhhiware) [inline plot_friends.py]: Please make it explicit that this is the number of days being printed
OR
just don't print maxdays

File: plot_friends.py [hunk 2]
```diff
@@ -86,5 +69,13 @@ def friends():
...
+    stats = []
+    stats.append(["Total Friends", cumulative_friends[-1]])
+    max_frnd_day = xaxis[frndcount.index(max(frndcount))]
+    stats.append(["Maximum Friends made on", max_frnd_day.date()])
+    stats.append(["Maximum Friends made in month", max_frnd_day.strftime('%B')])
```
Comment (kaustubhhiware) [inline plot_friends.py]: Let's say I made most friends on 28 Feb 2018.
I made the most friends in July.
Currently, the script returns `February`, I think it should return `July`.

File: plot_friends.py (hunk 1 above)
Comment (techytushar) [reply plot_friends.py]: ooh, I didn't mean to print it there, it was just for testing purpose. Forgot to delete it :sweat_smile:

File: plot_friends.py (hunk 2 above)
Comment (techytushar) [reply plot_friends.py]: sure !!

**Comment by techytushar** [comment]
Yes, we can make an examples folder wherein we can include some sample data, so that the user can also run it without using his own data.

**Pull request merged**